tools/validate/validate.cpp

```diff
@@ -19,15 +19,14 @@ constexpr size_t max_line_length = 120;
 
 class BinaryFile {
 public:
-    explicit BinaryFile(const filesystem::path& filepath) {
-        m_file = _wfopen(filepath.c_str(), L"rb");
+    explicit BinaryFile(const filesystem::path& filepath) : m_file(_wfopen(filepath.c_str(), L"rb")) {

```

Comment:
can you please add a line break before `` :`` so it can be read easier?

Reply:
clang-format says I cannot

Reply:
Well, you could try this:

```C++
  explicit BinaryFile(const filesystem::path& filepath)
  // clang-format off
  : m_file(_wfopen(filepath.c_str(), L"rb")) {
  // clang-format on
```

When using `// clang-format off`, the formatter interprets that as, "Don't do any formatting to code which appears before the next `// clang-format on` comment." Just remember that if you do this, then the formatter won't help you with, e.g., removing whitespace at the end of lines. Always run the `validate-binary.exe` tool after changes like this to make sure that the source file doesn't conflict with their style standards.